doc/Programming-guide/Using-Kaa-endpoint-SDKs/Objective-C/index.md

```diff
@@ -4,4 +4,109 @@ title: Objective-C
 permalink: /:path/
 sort_idx: 30
 ---
-Using Objective-C Kaa endpoint SDK
+
+**Table of Contents**
+
+* TOC
+{:toc}
+
+This guide describes how to use the KAA Objective-C SDK into your application, including base API overview and
+external links to demo applications and programming guides, which covered a pretty big amount of KAA architecture
+topics.
+
+# Requirements
+
+First, ensure that the following components are installed on your machine: 
+
+- Xcode 7.x or later 
+- CocoaPods 1.x.x or later
+
+# Installation
+
+The main method for installing Kaa SDK into a project as a third-party framework is using [CocoaPods](https://cocoapods.org/) dependency manager.
+
+First of all, before start to develop your application, you need to [generate Endpoint SDK](https://balldir.github.io/kaa/docs/current/Programming-guide/Using-Kaa-endpoint-SDKs/Java/Getting-started#generate-sdk) for the target platform - Objective-C. 
+After pressing on _download_ button, you'll get an archived file wrapped with __.tar.gz__ extension. 
+After downloading, please, move this archive into your project directory and extract.
+
+After that open a console, move into your project directory by _cd_ command. 
+When you're getting into required directory, write ``pod init`` into console in working directory or create a __Podfile__ manually, using your preferable text editor.
+
+If you create __Podfile__, using a command above, add a line to include Kaa SDK as pod:
+``` bash
+pod 'Kaa', :path => './libs/kaa-ep-sdk'
+```
+
+If you created a __Podfile__ by hand, first of all, you must fill it with as below:
+``` bash
+use_frameworks!
+
+target '<PROJECT_NAME>' do
+    pod 'Kaa', :path => './libs/kaa-ep-sdk'
+end
+```
+where __PROJECT_NAME__ - name of your project's target.
+
+Actually, you must add ``use_frameworks!`` command as first line into your __Podfile__, if you will use Kaa into swift project. 
+This step is mandatory if you don't want to bridge headers apart into your project.
+
+When you complete all steps before, complete next command chain:
+``` bash
+pod update 
+pod install
+```
+
+After that, you'll see a log message within extra frameworks success installation used in Kaa.
+If you don't get any errors, you can open your project by Xcode, running __xworkspace__ file.
+In your project tree you can find Kaa into __Development Pods__ folder, located  Pods target.
```

Comment:
Changed to the `group` instead of _directories_/_folders_, because of Xcode *.pbxproj project file stores information about code organization in special instances named groups.